src/Microsoft.Health.Fhir.Api/Features/Filters/ValidateExportRequestFilterAttribute.cs

```diff
@@ -32,6 +32,7 @@ public ValidateExportRequestFilterAttribute()
             _supportedQueryParams = new HashSet<string>(StringComparer.Ordinal)
             {
                 KnownQueryParameterNames.Since,
+                KnownQueryParameterNames.Type,
```

Comment:
nit: We have some UTs in `ValidateExportRequestFilterAttributeTests`. Might be useful to add one for this scenario

Reply:
Added